transport/ax25/ax25.go

```diff
@@ -124,18 +129,22 @@ func (d Dialer) DialURL(url *transport.URL) (net.Conn, error) {
 	case "ax25":
 		return DialAX25Timeout(url.Host, url.User.Username(), target, d.Timeout)
 	case "serial-tnc":
-		//TODO: This is some badly designed legacy stuff. Need to re-think the whole
-		//serial-tnc scheme. See issue #34.
+		// TODO: This is some badly designed legacy stuff. Need to re-think the whole
+		// serial-tnc scheme. See issue #34.
 		baudrate := Baudrate(1200)
```

Comment:
Do you want to take the opportunity to distinguish this as `rfBaud` or `hBaud`?

Reply:
Good point! Thank you 🙂